configure.ac

```diff
@@ -487,34 +487,35 @@ AS_IF([test x$have_gui = xyes],
 [
   # settings for Qt
   LIBS="$LIBS $QT_LIBS"
-  PKG_FLAGS="$PKG_FLAGS $QT_CFLAGS"
+  # add -fPIC flag to satisfy qt on linux
+  AC_MSG_CHECKING([Checking whether we have to add -fPIC flag to make Qt happy - ])
+  CPPFLAGS_backup="$CPPFLAGS"
+  CPPFLAGS="$CPPFLAGS $QT_CFLAGS"
+  AC_COMPILE_IFELSE([AC_LANG_PROGRAM(
+  [
+    #include <QtCore/QObject>
+  ], [])], AC_MSG_RESULT([no]),
+  [
+    AC_MSG_RESULT([yes])
+    PKG_FLAGS="$PKG_FLAGS -fPIC"
+  ])
+  CPPFLAGS="$CPPFLAGS_backup"
   MOCFLAGS="$QT_CFLAGS"
   AS_IF([which moc 2>&1 >/dev/null],
   [
     QTMOC="moc"
   ],
   [
-    QTMOC=`$PKG_CONFIG --variable=moc_location QtCore`
+    QTMOC=`$PKG_CONFIG --variable=moc_location Qt5Core`
   ])
-  QTLIBDIR=`$PKG_CONFIG --variable=libdir QtGui`
+  QTLIBDIR=`$PKG_CONFIG --variable=libdir Qt5Gui`
 
   AC_SUBST(QTMOC)
   AC_SUBST(QTLIBDIR)
   AC_SUBST(MOCFLAGS)
 
-  # check if floating control panel has to be forced
-  PKG_CHECK_MODULES([QtGui_4_7_or_higher], [QtGui >= 4.7.0],
-  [
-    AS_IF([test x$enable_floating_control_panel = xno],
-    [
-      AC_MSG_ERROR([With Qt >= 4.7 the floating control panel is obligatory!])
-    ],
-    [test x$enable_floating_control_panel = x],
-    [
-      AC_MSG_WARN([Found Qt >= 4.7, forcing --enable-floating-control-panel!])
-      enable_floating_control_panel=yes
-    ])
-  ], [true])
+  # enable floating control panel, as it is the only option on qt5
+  enable_floating_control_panel=yes
```

Comment:
The option `--disable-floating-control-panel` doesn't work anymore (it should cause an error with Qt5).

I guess we will never use the SSR with Qt < 4.7 again, right?
Then we should get rid of the whole floating control panel option!